_config.yml

```diff
@@ -9,11 +9,11 @@ url: https://theforeman.org
 baseurl: "" #keep at "" for the Atom feed
 
 # latest *stable* version
-foreman_version: '3.7'
+foreman_version: '3.8'
 
 # Latest releases
 foreman_latest_releases:
-  - minmaj: '3.7'
+  - minmaj: '3.8'
     patch: '0'
   - minmaj: '3.6'
```

Comment:
You need to modify this to 3.7. The result should be on https://theforeman.org/ under "Latest versions".